src/engraving/dom/harmony.h

```diff
@@ -201,6 +198,21 @@ class Harmony final : public TextBase
     bool isDrawEditMode() const { return m_isDrawEditMode; }
     void setIsDrawEditMode(bool val) { m_isDrawEditMode = val; }
 
+    struct LayoutData : public TextBase::LayoutData {
+        bool isSetHarmonyHeight() const { return m_harmonyHeight.has_value(); }
```

Comment:
Seems unused; is it intentional that it's still added?

Reply:
It is added for a unified approach, I think it would be good practice for all such methods to have a setter, a getter and a check for what is set.